testing/kuttl/e2e/pgbouncer/00-cluster.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: pgbouncer-test
+spec:
+  postgresVersion: 14
+  instances:
+    - name: instance1
+      replicas: 2
```

Comment:
Why have two replicas :question: It doesn't look like it is necessary as part of the pgBouncer checks. Can we reduce it to 1 to make the tests a little lighter?